src/test/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/integration/PrisonerSearchResourceIntTest.kt

```diff
@@ -129,15 +128,15 @@ class PrisonerSearchResourceIntTest : IntegrationTestBase() {
 
     @Test
     fun `should search by age with one term`() {
-      val age = LocalDate.now().year - LocalDate.parse("1987-01-01").year
+      val age = LocalDate.now(fixedClock).year - LocalDate.parse("1987-01-01").year
```

Comment:
could hard code the `age` since it will always be the same (as based off the fixed clock).  It makes more sense to me to leave it in so that calculation of the age is shown too